src/AspNetCore/Src/applicationinfo.cpp

```diff
@@ -264,16 +264,21 @@ APPLICATION_INFO::FindRequestHandlerAssembly()
             goto Finished;
         }
 
-        if (FAILED(hr = FindNativeAssemblyFromHostfxr(&struFileName)))
+        if (m_pConfiguration->QueryHostingModel() == APP_HOSTING_MODEL::HOSTING_IN_PROCESS)
```

Comment:
we need log a windows event in case of failure. Nothing was logged currently

Reply:
done.

Reply:
this is critical error. should be logged as error instead of info.
please also make sure this log is not pre request. It should be logged only once per applicationinfo

Reply:
It can't be logged more than once because we set g_fAspnetcoreRHLoadedError if this function failed.